users/assets/login_authenticate.py

```diff
@@ -0,0 +1,13 @@
+from django.shortcuts import redirect
```

Comment:
@ahmadx16 Lets name this file utils.py and keep it under users directory, no need for assets directory/package.